File: Python/Conditional Statement/bmi.py
w = float(input("Enter weight in kg: "))
h = float(input("Enter height in cm: "))

hm = h / 100
bmi = w / (hm * hm)

print(f"Your BMI is: {bmi}")

if bmi < 18.5:
    print("Underweight")
elif bmi >= 18.5 and bmi < 25:
    print("Normal Weight")
elif bmi >= 25 and bmi < 30:
    print("Overweight")
elif bmi >= 30 and bmi < 35:
    print("Obese (Class I)")
elif bmi >= 35 and bmi < 40:
    print("Obese (Class II)")
else:
    print("Extreme Obesity (Class III)")

File: C/Introduction/bmi.c
#include<stdio.h>

int main(){
	
	float weight, height, height_m, bmi;
	
	printf("Please enter your Weight(kg) and Height(cm):");
	scanf("%f %f",&weight, &height);
	
	height_m = height/100;
	
	bmi = weight/(height_m*height_m);
	
	
	printf("BMI = %f",bmi);
	return 0 ;
}
